src/components/Icon/Icon.css

```diff
@@ -1,32 +1,128 @@
 .Icon {
+  --icon-margin: 10px;
+
+  &--small {
+    --icon-size: 24px;
+  }
+
+  &--medium {
+    --icon-size: 36px;
+  }
+
+  &--large {
+    --icon-size: 48px;
+  }
+
+  background: #fff;
   content: "";
-  display: block;
   display: inline-block;
-  height: 24px;
-  margin-right: 10px;
+  height: var(--icon-size);
   mask-position: center;
   mask-repeat: no-repeat;
   mask-size: contain;
   position: relative;
-  width: 24px;
+  width: var(--icon-size);
 
+  /* Alignment */
```

Comment:
Nope. Add margins via css, not markup or props.

Reply:
https://github.com/adaptdk/su-ukraina/pull/219/commits/314165ea7996c34a2fa7c2b7c27783154ce6f11b